docs/dev_guides/manipulators.md

```diff
@@ -4,19 +4,22 @@ Manipulators are UI elements attached to napari layers to aid with positioning a
 being displayed on the canvas. Manipulators can translate and rotate along/around specified axes.
 
 ## Coordinate system
-The manipulator coordinate system is defined with an affine transformation relative to the layer data coordinate system.
-The manipulator transformation is defined as a translation and a 3x3 transformation matrix (rotation and scale). The 
-translation is stored in the `manipulator.translation` property and the transformation is stored in the `manipulator.
-rot_mat` property. The rotation is applied before the translation. 
+The manipulator coordinate system is the napari `world` coordinate system (for more information see [the napari
+ documentation](https://napari.org/stable/guides/3D_interactivity.html#coordinate-systems-in-napari)).
+The manipulator transformation is defined as a translation (of the origin of the manipulator) and a 3x3 transformation
```

Comment:
```suggestion
The manipulator transformation is an affine transformation; defined as a translation (of the origin of the manipulator) and a 3x3 transformation
```

Reply:
Is it a good idea to keep "affine" in the text?

Reply:
I think this is more clear. Thanks @GenevieveBuckley !